include/swift/AST/Attr.def

```diff
@@ -502,6 +502,12 @@ SIMPLE_DECL_ATTR(_nonEphemeral, NonEphemeral,
   ABIStableToAdd | ABIStableToRemove | APIBreakingToAdd | APIStableToRemove,
   90)
 
+DECL_ATTR(differentiable, Differentiable,
+  OnAccessor | OnConstructor | OnFunc | OnVar | OnSubscript | LongAttribute |
+  AllowMultipleAttributes |
+  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
```

Comment:
I believe this affects ABI, so I'd recommend removing the  "`ABIStableToAdd | ABIStableToRemove`" tags, as well as `APIStableToRemove` which seems inappropriate.

Reply:
It's not clear to me what these tags mean exactly, but I'll try the following configuration on `tensorflow` branch to see if things break:

```
ABIBreakingToAdd | ABIBreakingToRemove | APIStableToAdd | APIBreakingToRemove
```

Testing on `tensorflow` branch in https://github.com/apple/swift/pull/28148. The PR description has more context.

Reply:
Replaced the attributes to be similar to `tensorflow` branch.